index.js

```diff
@@ -16,16 +16,25 @@ app.use(route.get('/db', initDB));
 app.use(route.get('/register', register));
 
 function *index() {
-  console.log(this.req);
   this.body = {'message': 'Hello world'};
 }
 
 function *initDB() {
   yield this.pg.db.client.query_('CREATE EXTENSION postgis;');
-  yield this.pg.db.client.query_('CREATE TABLE "users" ("id" SERIAL, "uuid" UUID NOT NULL, PRIMARY KEY ("id"));');
-  yield this.pg.db.client.query_('CREATE TABLE "check_ins" ("id" serial, "users.id" serial, PRIMARY KEY ("id"), FOREIGN KEY ("users.id") REFERENCES "users"("id") ON DELETE CASCADE ON UPDATE CASCADE);');
-  yield this.pg.db.client.query_('SELECT AddGeometryColumn(\'check_ins\', \'location\', 4326, \'POINT\', 2);');
-  this.body = '1'
+  yield this.pg.db.client.query_('CREATE TABLE "users" ("id" SERIAL, "uuid" UUID NOT NULL, PRIMARY KEY ("id"), UNIQUE ("uuid"));');
+  yield this.pg.db.client.query_('CREATE TABLE "places" ("id" serial, "name" text, "guid" UUID NOT NULL, PRIMARY KEY ("id"), UNIQUE ("guid"));');
+  yield this.pg.db.client.query_('SELECT AddGeometryColumn(\'places\', \'coordinate\', 4326, \'POINT\', 2);');
```

Comment:
Line is too long.<br>'yield' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).